### ISSUE TITLE
CaPTk 1.7.6 GUI Failure

### ISSUE THREAD

**Issue by ShyamDesai** [CBICA/CaPTk#876]
**Describe the bug**
Failed to load CaPTk GUI from cluster. Crashes immediately upon execution of 'captk' in CL. Have tried to unload captk before loading the new version as well.

**To Reproduce**
1. On the cluster
2. "Module load captk/1.7.6"
3. "captk"

**Expected behavior**
The CaPTk GUI should open.

**Screenshots**
This is the error I am receiving:
Checking for compatible OpenGL - this will happen only once.
WARNING: Trying to run CaPTk GUI using software rendering - this might not work on all systems and in those cases, only the CLI will be available.
[0220/112554.064064:WARNING:stack_trace_posix.cc(699)] Failed to open file: /scratch/desaish/#9175046 (deleted)
  Error: No such file or directory
 ApplicationPreferences::DisplayPreferences()
 font =  ""
 theme =  ""
 ApplicationPreferences::DisplayPreferences()
 font =  "Sans Serif,9,-1,5,50,0,0,0,0,0"
 theme =  "Dark"
 ApplicationPreferences::DisplayPreferences()
 font =  "Sans Serif,9,-1,5,50,0,0,0,0,0"
 theme =  "Dark"
p11-kit: couldn't list directory: /etc/pki/ca-trust/source/anchors: Permission denied
The X11 connection broke: I/O error (code 1)
XIO:  fatal IO error 0 (Success) on X server "170.166.98.52:17.0"
      after 720 requests (720 known processed) with 0 events remaining.

**CaPTk Version**
New 1.7.6

**Desktop (please complete the following information):**
Windows 10 Enterprise
Version 1809
OS Build 17763.973

**Additional context**
I have tried going through the captk path which I believe is: /cbica/software/lab/captk/centos7/1.7.6/bin

From here I am able to access specific CaPTk software like FeatureExtraction/BreastTexturePipeline but so far have not been able to access the GUI for 1.7.6

**Comment by ashishsingh18** [comment]
Related to #831, #874, #875

**Comment by dboun** [comment]
Duplicate of #831
